optaplanner-examples/src/main/resources/org/optaplanner/examples/meetingscheduling/solver/meetingSchedulingScoreRules.drl

```diff
@@ -75,7 +66,7 @@ rule "Start and end on same day"
         MeetingAssignment(startingTimeGrain != null, $firstTimeGrain : startingTimeGrain, $lastTimeGrainIndex : getLastTimeGrainIndex())
         $lastTimeGrain : TimeGrain(grainIndex == $lastTimeGrainIndex, $firstTimeGrain.getDay() != getDay())
     then
-        scoreHolder.addHardConstraintMatch(kcontext, -1);
+        scoreHolder.addHardConstraintMatch(kcontext, - MeetingSchedulingConstraints.START_AND_END_ON_SAME_DAY.getConstraintWeight());
```

Comment:
Use the Parameterization pattern (naming, how it comes from the data file, etc) as done in
  https://github.com/kiegroup/optaplanner/blob/master/optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/domain/ConferenceParametrization.java

That also gets loaded from the Excel file.

Other examples doing it:
https://github.com/kiegroup/optaplanner/blob/master/optaplanner-examples/src/main/java/org/optaplanner/examples/rocktour/domain/RockTourParametrization.java
https://github.com/kiegroup/optaplanner/blob/master/optaplanner-examples/src/main/java/org/optaplanner/examples/investment/domain/InvestmentParametrization.java
etc

Motivation 1: by using the same naming convention (*Parameterization), it's easier for users to recognize it and for us to document it
Motivation 2: This PR kills multitenancy: if 2 solvers run a meeting scheduling example at the same time on a different dataset and both have a different parametrization, this is a bug, because their weights are stored in a single static field per constraint rule.